src/moment-options.ts

```diff
@@ -0,0 +1,16 @@
+import { InjectionToken } from '@angular/core';
+
+export const NGX_MOMENT_OPTIONS: InjectionToken<string> = new InjectionToken<string>('NGX_MOMENT_OPTIONS');
```

Comment:
According to docs, it seems like the type parameter for `InjectionToken` should be the interface name (i.e. `NgxMomentOptions`):

https://angular.io/api/core/InjectionToken

Reply:
Yep, my bad and good spot, updated in next commit.